src/test/kotlin/no/fdk/dataset/preview/service/PreviewContractTest.kt

```diff
@@ -31,11 +29,11 @@ class PreviewContractTest : ApiTestContext() {
 
     @Test
     fun `Unauthorized when api token is not included`() {
-        val previewRequest = PreviewRequest("http://localhost:5000/download", 5)
-        val rsp = authorizedRequest("/preview", port, mapper.writeValueAsString(previewRequest),
-            null, HttpMethod.POST)
-
-        assertEquals(HttpStatus.FORBIDDEN.value(), rsp["status"])
+        assertThrows<RuntimeException> {
```

Comment:
betyr RuntimeException at du har fått HttpStatus = Forbidden?

Reply:
Det gjelder kun i testen. RuntimeException betyr at man ikke har klart å få tak i csrf token.

Reply:
Ah, ok. Kanskje det hadde vært triks å gjøre noe som tydeliggjør det? Kanskje assert på at feilmeldingen inneholder `Unable to fetch CSRF token`?